sale_order_line_sequence/views/sale_view.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
don't do this in xml files

Reply:
Can you please, tell me why it's necessary in XML flies? 

thank for review.

Reply:
because we follow our guidelines https://github.com/OCA/maintainer-tools

An example here: https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml

Reply:
thank you, fixed follow template.